package.json

```diff
@@ -25,7 +25,7 @@
     "parser": "babel-eslint"
   },
   "engines": {
-    "node": ">=16"
+    "node": ">=17"
```

Comment:
Isn't it 21 we require elsewhere in the repo?

Reply:
Yes but 17 is the minimum